### PR TITLE
my cv

### PR DESCRIPTION
**Pull request by carmecass**
My cv

### CODE REVIEW

File: staff/carme-casserras/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>CV CARMECASS</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="css/style.css" rel="stylesheet">
```
Comment (lexbarba) [inline staff/carme-casserras/cv/index.html]: @carmecass There should we no link to a css file as this project is pure html. Please remove.

File: staff/carme-casserras/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                            <a href="carmecass@gmail.com">carmecass@gmail.com</a>
+                        </li>
+                        <li>
+                            <a href="https://www.linkedin.com/messaging/compose/?after=profile.view&afterParam=carme-casserras-736b06b&body=https%3A%2F%2Fwww.linkedin.com%2Fin%2Fcarme-casserras-736b06b&lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base%3BB2x30DFsSqG2VUD%2B3br0KA%3D%3D&licu=urn%3Ali%3Acontrol%3Ad_flagship3_profile_view_base-message">Linkedin</a>
+                        </li>
+                    </ul>
+                </nav>
+            </section>
+            <aside>
+                <img src="https://ca.slack-edge.com/T0SJKHBFZ-UEETBFA2W-b9014c205756-72">
```
Comment (lexbarba) [inline staff/carme-casserras/cv/index.html]: @carmecass "alt" attribute is highly recommended in all img tags. Please check in the following link: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img .

File: staff/carme-casserras/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            </aside>
+    </header>
+        <main>
+            <section>
+                <h3>EDUCACION</h3>
+                    <ul>
+                        <li>1.992-1.994 I.E.S.E. UNIVERSIDAD DE NAVARRA</li>
+                            <p>Master en Economía y Dirección de Empresas</p>
```
Comment (lexbarba) [inline staff/carme-casserras/cv/index.html:39]: @carmecass np <p> should be nested inside <ul>. you can put inside <li> element any html element, but inside a <li>, not outisde. <ul> should only contain <li> elements.

**Review by lexbarba — CHANGES_REQUESTED** [review]

File: staff/carme-casserras/cv/index.html (hunk 1 above)
Comment (carmecass) [reply staff/carme-casserras/cv/index.html]: Done!

Comment (robert-z) [reply staff/carme-casserras/cv/index.html]: Yep! You needed to deleted only de `link` element, not all `head` element 😄

**Pull request merged**